### ISSUE TITLE
How to not get arrested in Japan

### ISSUE THREAD

**Issue by okonomiyaki3000** [hamukazu/lets-get-arrested#22]
One time, we were having lunch in Roppongi in a place overlooking a the street. We saw 4 extremely drunk (it was not quite noon) 20-somethings emerge from somewhere and proceed to cross the street. 2 girls and one guy made it across but the other guy loitered in the middle even as the lights changed. As taxis drove past him, he kicked at them. Soon he was pissing in the middle of the street. Meanwhile, two of his friends were rolling around on the ground on the side of the road. This went on for 10 or 15 minutes. While it was happening, a patrol car came and parked on the side of the road nearby for a minute or two and then pulled away without incident, two cops on foot stopped on the sidewalk and watched with disapproval on their faces and then left, and one motorcycle cop did a U-turn right in front of the guy but was met only with aggressive body language. Eventually the scene cleared up on its own without any arrests or citations.

**Comment by kujiy** [comment]
Because they didn't know this trick ;)

**Comment by hamukazu** [comment]
Yes. According to the police in Japan, writing an infinite loop in JavaScript is  a more serious crime than kicking taxis and pissing in the middle of street.

**Comment by ghost** [comment]
Yes, because code failures can cause loss of unique data.
